testdata/scripts/literals-all-obfuscators.txt

```diff
@@ -0,0 +1,27 @@
+generate-literals main.go 500
+
+go build
+exec ./main$exe
+
+cp stderr normal.stderr
+
+garble -literals -debugdir=.obf-src -seed=8J+Ri/Cfh6fwn4e+ build
```

Comment:
I apologise for not bringing this up earlier, but please try to add to an existing test file; in particular, literals.txt seems like a good place. For the reasoning behind that, see https://github.com/mvdan/garble/pull/79#discussion_r462241648.